### PR TITLE
Fix broken training code

### PR DESCRIPTION
**Pull request by SoonminHwang**
이전 PR(#3) 때문에 망가진 학습 코드를 수정하기 위한 PR입니다.
지금 제가 사용하던 머신에 문제가 있어서 학습코드가 동작하는 것만 확인했습니다.
학습 코드 자체에서 큰 변화는 없기때문에 성능 차이는 없을 것이라고 생각하지만 확인 부탁합니다.
@socome @xown3197 @rgw117

### CODE REVIEW

**Comment by SoonminHwang** [comment]
KASIT_BENCHMARK

batch_size를 12로 늘려서 그런지 성능은 오히려 조금 더 좋아지는걸로.. 어쨌든 학습코드까지 잘 돌아가는거 확인했습니다.

**Comment by wlc-git** [comment]
안녕하세요
테스트를 위해 그림을 몇 개 사용하고 싶은데, 테스트 후에 사용 사례 그림을 어떻게 얻습니까?당신의 회답을 기대합니다.감사합니다!

**Comment by socome** [comment]
Could you please leave this comment in issue, and write this in English so that others also can understand it. @wlc-git

**Comment by wlc-git** [comment]
Hello:
I want to use the test picture to test, how can I get the visual result of the test case? I look forward to your reply. Thank you

**Comment by xown3197** [comment]
Check out https://github.com/sejong-rcv/MLPD-Multi-Label-Pedestrian-Detection/pull/5
@wlc-git
